docs/user-guide/cli-using-team-managing-credential-security.md

```diff
@@ -1,15 +1,77 @@
+Secure credential encryption is now included with the Zowe CLI core application. When a command using a profile with missing `user` and `password` information is issued, Zowe CLI V2 prompts you to enter the username and password by default. To save secure properties and values in the Secure Credential Store, use the `zowe config` command group.
 
-You can use an editor to define options to the secure array in `zowe.config.json`. Any option that you define to there becomes secure/prompted-for.
+## Configuring secure properties
 
-## Changes to secure credential storage
+Create a configuration file and set its secure properties (such as usernames and passwords):
 
-With the introduction of team profiles in Zowe CLI V2, the **Secure Credential Store (SCS) Plug-in** is deprecated. The `zowe scs` command group is obsolete. Secure credential encryption is now included with the Zowe CLI core application.
+1. Open the Zowe CLI command prompt.
 
-Zowe CLI V2 prompts you to enter the `username` and `password` securely by default. Commands in the `zowe config` command group let you manage security for any option value.
+2. To initialize a **project team** configuration file in the current working directory:
+
+    ```
+    zowe config init
+    ```
+    To initialize a **project user** configuration file in the current working directory:
+    ```
+    zowe config init --user-config
+    ```
+    To initialize a **global team** configuration file in the `ZOWE_CLI_HOME` directory:
+    ```
+    zowe config init --global-config
+    ```
+    To initialize a **global user** configuration file in the `ZOWE_CLI_HOME` directory:
+
+    ```
+    zowe config init --global-config --user-config
+    ```
+    A configuration file is created, if one does not already exist.
+    
+    Additionally, the `profiles.base.properties.user` and `profiles.base.properties.password` fields are added to the base profile `secure` array for that configuration file. This stores the username and password in the Secure Credential Store.
+
+3. If needed, add other fields to the secure array.
+    - Use a text editor or an IDE (such as Visual Studio Code) to edit the configuration file.
+    - Issue the `zowe config set --secure <property-path>` command to secure a specific property in a specific profile.
+        
+        For example, `zowe config set profiles.base.properties.password pw123 --secure` adds the `password` property to the base profile's `secure` array and saves the password `pw123` in the Secure Credential Store.
+
+        If you issue the command for a property that is already secured, the CLI prompts you to enter a new property value.
+
+    The values for these properties are saved in the Secure Credential Store.
+
+## Updating secure properties
```

Comment:
Change to "Update secure properties"

Reply:
FYI: We use gerunds for procedures/actions. See the Zowe Docs [style guide](https://docs.zowe.org/stable/contribute/contributing#for-tasks-and-procedures-use-gerunds-for-headings).